Dockerfile

```diff
@@ -3,7 +3,7 @@
 FROM alpine
 
 # Enable HTTPS support in wget.
-RUN apk add --update openssl
+RUN apk add --update --no-cache openssl bash
```

Comment:
I'm not sure I understand how `apk` works, but running with `--no-cache` means there's no cache to clean up. Are you proposing to do that instead of `--no-cache`?

Reply:
Hmm, good question. I just ran `apk cache clean` in a pristine Alpine image, and it said `ERROR: Package cache is not enabled.`. Sp that command clearly makes no sense here.

Now I wonder whether `--no-cache` makes any difference?